lib/iiif_print/split_pdfs/adventist_pages_to_jpgs_splitter.rb

```diff
@@ -17,7 +17,7 @@ module AdventistPagesToJpgsSplitter
       #
       # @see https://github.com/scientist-softserv/iiif_print/blob/a23706453f23e0f54c9d50bbf0ddf9311d82a0b9/lib/iiif_print/jobs/child_works_from_pdf_job.rb#L39-L63
       def self.call(path,
-                    splitter: DerivativeRodeoSplitter,
+                    splitter: Rails.env.development? ? PagesToJpgsSplitter : DerivativeRodeoSplitter,
```

Comment:
another thought is if this is because we don't have credentials, then instead of assuming development doesn't have credentials, maybe we can check if we have credentials to begin with

Reply:
consider switching using an env var instead